models/shipping-method/src/zone-rate/transformers.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  KeyReference,
+  Reference,
+  TReferenceGraphql,
+} from '@commercetools-test-data/commons';
+import { Transformer } from '@commercetools-test-data/core';
+import type { TZoneRate, TZoneRateGraphql, TZoneRateRest } from './types';
+
+const transformers = {
+  default: Transformer<TZoneRate, TZoneRate>('default', {
+    buildFields: ['zone', 'shippingRates'],
+  }),
+  rest: Transformer<TZoneRate, TZoneRateRest>('rest', {
+    buildFields: ['zone', 'shippingRates'],
+    replaceFields: ({ fields }) => ({
+      ...fields,
+      zone: KeyReference.random().typeId(fields.zone.id).buildRest(),
```

Comment:
```suggestion
      zone: KeyReference.random().id(fields.zone.id).typeId('zone').buildRest(),
```

Reply:
`id` throws an error here - there is no id on the KeyReference object

Reply:
That or use they `key` from the field